android/com/facebook/buck/android/support/exopackage/ExopackageApplication.java

```diff
@@ -162,4 +162,15 @@ public void onConfigurationChanged(Configuration newConfig) {
       delegate.onConfigurationChanged(newConfig);
     }
   }
+
+  @Override
+  public Object getSystemService(String name) {
+    if (delegate != null) {
+      Object service = delegate.getSystemService(name);
```

Comment:
do you want to check if name is not null here? given that we may not be adding the annotation?